packages/synapse-react-client/src/components/dataaccess/UseAccessRequirementTable.tsx

```diff
@@ -307,6 +293,20 @@ export function useAccessRequirementTable(
         sorting: sortingState,
         columnFilters: columnFilters,
       },
+      getFacetedUniqueValues: (
+        table: Table<AccessRequirementSearchResult>,
+        columnId: string,
+      ) => {
+        if (columnId === 'type') {
+          // Hard-code the set of possible access requirement types with count = 0 to indicate that we don't have facet counts.
+          return () =>
+            new Map(ALL_ACCESS_REQUIREMENT_CONCRETE_TYPES.map(v => [v, 0]))
+        }
```

Comment:
We want to provide faceted filter options, where we know all of the facet values, but we don't have facet counts. In this case, we provide a custom implementation that returns all of the facet values with counts set to 0.